<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" tests="7" skipped="0" failures="1" errors="0" timestamp="2024-03-14T17:57:02" hostname="MUNIT-914" time="0.005">
  <properties/>
  <testcase name="first test with higher priority" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.0"/>
  <testcase name="second test with higher priority" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.0"/>
  <testcase name="testFahrenheitToCelsiusConversion[0](32.0, 0.0)" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.001"/>
  <testcase name="testFahrenheitToCelsiusConversion[1](212.0, 100.0)" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.0"/>
  <testcase name="testFahrenheitToCelsiusConversion[2](-40.0, -40.0)" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.0"/>
  <testcase name="testFahrenheitToCelsiusConversion[3](98.6, 37.0)" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.0"/>
  <testcase name="testFahrenheitToCelsiusConversion[4](-1111.0, -1111.0)" classname="big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest" time="0.003">
    <failure message="java.lang.AssertionError: Conversion from Fahrenheit to Celsius is incorrect expected [-1111.0] but found [-635.0]" type="java.lang.AssertionError">java.lang.AssertionError: Conversion from Fahrenheit to Celsius is incorrect expected [-1111.0] but found [-635.0]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEquals(Assert.java:698)
	at big.pckg.name.fooor.failing.testsss.added.solely.inn.verification.FahrenheitToCelsiusConversionTest.testFahrenheitToCelsiusConversion(FahrenheitToCelsiusConversionTest.kt:12)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:819)
	at org.testng.TestRunner.run(TestRunner.java:619)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
	at org.testng.SuiteRunner.run(SuiteRunner.java:336)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
	at org.testng.TestNG.runSuites(TestNG.java:1134)
	at org.testng.TestNG.run(TestNG.java:1101)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:148)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:95)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:62)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
